wgpu/src/util/init.rs

```diff
@@ -37,13 +37,13 @@ pub fn power_preference_from_env() -> Option<PowerPreference> {
 
 /// Initialize the adapter obeying the WGPU_ADAPTER_NAME environment variable.
 #[cfg(not(target_arch = "wasm32"))]
-pub fn initialize_adapter_from_env(instance: &Instance, backend_bits: Backends) -> Option<Adapter> {
+pub fn initialize_adapter_from_env(instance: &Instance) -> Option<Adapter> {
     let desired_adapter_name = std::env::var("WGPU_ADAPTER_NAME")
         .as_deref()
         .map(str::to_lowercase)
         .ok()?;
 
-    let adapters = instance.enumerate_adapters(backend_bits);
+    let adapters = instance.enumerate_adapters(Backends::all());
```

Comment:
it took me really long to understand that `enumerate_adapter` does in fact not list backends that it didn't know about at creation. Should really be explicit on on its doc string I suppose.